pandas/tseries/tools.py

```diff
@@ -238,6 +238,18 @@ def to_datetime(arg, errors='raise', dayfirst=False, yearfirst=False,
         datetime strings, and if it can be inferred, switch to a faster
         method of parsing them. In some cases this can increase the parsing
         speed by ~5-10x.
+    origin : scalar convertible to Timestamp / string ('julian', 'epoch'),
+        default 'epoch'.
+        Define relative offset for the returned dates.
+
+        - If 'epoch', offset is set to 1970-01-01.
```

Comment:
I would maybe use `origin` instead of `offset` (here and in lines below). The keyword is named like that, and I also think it is more correct word to use. And offset is something like a 'timedelta', while the origins here are actual 'timestamps'